adhocteam/Head-Start-TTADP
# Add frontend file upload

Changes styling to match design for https://github.com/HHS/Head-Start-TTADP/issues/150
I have connected the uploading 

**How to test**
Pull down changes
Add a grantee to the activity report
click on topics and resources
add a file to either resources or attachments
Check minio for the uploaded file.
Check the database for proper metadata





**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/150

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
